system/jlib/jlog.hpp

```diff
@@ -796,23 +806,23 @@ extern jlib_decl ILogMsgHandler * attachLogMsgMonitorFromPTree(IPropertyTree * t
 extern jlib_decl void attachManyLogMsgMonitorsFromPTree(IPropertyTree * tree);            // Takes tree containing many <monitor> elements
 
 // Standard categories and unknown jobInfo
-constexpr LogMsgCategory MCdisaster(MSGAUD_all, MSGCLS_disaster);
-constexpr LogMsgCategory MCuserError(MSGAUD_user, MSGCLS_error);
-constexpr LogMsgCategory MCoperatorError(MSGAUD_operator, MSGCLS_error);
-constexpr LogMsgCategory MCinternalError(MSGAUD_programmer, MSGCLS_error, 1);
-constexpr LogMsgCategory MCauditError(MSGAUD_audit, MSGCLS_error);
-constexpr LogMsgCategory MCuserWarning(MSGAUD_user, MSGCLS_warning);
-constexpr LogMsgCategory MCoperatorWarning(MSGAUD_operator, MSGCLS_warning);
-constexpr LogMsgCategory MCinternalWarning(MSGAUD_programmer, MSGCLS_warning, 1);
-constexpr LogMsgCategory MCauditWarning(MSGAUD_audit, MSGCLS_warning);
-constexpr LogMsgCategory MCuserProgress(MSGAUD_user, MSGCLS_progress);
-constexpr LogMsgCategory MCoperatorProgress(MSGAUD_operator, MSGCLS_progress);
-constexpr LogMsgCategory MCdebugProgress(MSGAUD_programmer, MSGCLS_progress);
-constexpr LogMsgCategory MCuserInfo(MSGAUD_user, MSGCLS_information);
-constexpr LogMsgCategory MCdebugInfo(MSGAUD_programmer, MSGCLS_information);
-constexpr LogMsgCategory MCauditInfo(MSGAUD_audit, MSGCLS_information);
-constexpr LogMsgCategory MCstats(MSGAUD_operator, MSGCLS_progress);
-constexpr LogMsgCategory MCoperatorInfo(MSGAUD_operator, MSGCLS_information);
+constexpr LogMsgCategory MCdisaster(MSGAUD_all, MSGCLS_disaster, FatalMsgThreshold);
+constexpr LogMsgCategory MCuserError(MSGAUD_user, MSGCLS_error, ErrMsgThreshold);
+constexpr LogMsgCategory MCoperatorError(MSGAUD_operator, MSGCLS_error, ErrMsgThreshold);
+constexpr LogMsgCategory MCinternalError(MSGAUD_programmer, MSGCLS_error, AllMsgThreshold);
+constexpr LogMsgCategory MCauditError(MSGAUD_audit, MSGCLS_error, ErrMsgThreshold-10);
+constexpr LogMsgCategory MCuserWarning(MSGAUD_user, MSGCLS_warning, WarnMsgThreshold+5);
```

Comment:
as above, it's a warning, so think clearer if has same threshold, and can already filter if wanted by audience.

Reply:
well... we have thresholds, and windows in between those thresholds. Some times the category doesn't belong at the base level. You might be right about this particular category, but in general designating a category to be slightly above a particular threshold is fine. Otherwise, the range should be 1-10 (which it prob should be)

Reply:
there's perhaps no harm differentiating, but I'm not sure it's clear to someone who might configure the system why these differ, where as audience, e.g. audit , internal, user do carry more meaning.
I'd make them the same, and if we ever wanted to differentiate, then allow e.g. "user" audience messages to have a different logging level.

Reply:
I agree with @jakesmith  on using the set threshold levels (i.e. using WarnMsgThreshold)